app/assets/stylesheets/components/_taxon-list.scss

```diff
@@ -0,0 +1,13 @@
+.app-c-taxon-list__item {
+  @include core-19;
+  list-style: none;
+  margin-bottom: $gutter-two-thirds;
+}
+
+.app-c-taxon-list__heading {
+  margin-bottom: $gutter-one-third / 2;
```

Comment:
I wondered the same, but kept as-is for purposes of making the component.

Reply:
Fair enough. My only other thought on that (which isn't applicable in this case anyway) is that dividing variables like this could result in a non-round number, at which point browsers may return single pixel variations depending on which way they decide to go. However, most of the time this wouldn't be a problem.